common/test/acceptance/tests/lms/test_lms.py

```diff
@@ -283,6 +305,14 @@ def test_deferred_verification_enrollment(self):
         # Submit payment
         self.fake_payment_page.submit_payment()
 
+        # Expect enrollment activated event
+        assert_event_emitted_num_times(
```

Comment:
Hmm. Still looking at the other tests, but in this test in particular, the enrollment.activated event is not found until this point (i.e. after submitting payment). I even put in 2-second sleeps earlier in the test, but this is the soonest that event is emitted.